_inc/client/plans/single-product-backup.scss

```diff
@@ -125,6 +138,34 @@ input[type='radio'].plan-radio-button__input {
 }
 
 .single-product-backup__accented-card-body p {
-	margin-top: 16px;
+	margin-top: 14px;
+	font-size: 14px;
+	color: $gray-text-min;
+	text-align: center;
+}
+
+.single-product-backup__promo {
+	background-color: #f8f8f8;
```

Comment:
Should we have the color here and below be pulled from a variable?

Reply:
Sure! Should I add the variable to the global `colors.scss`, or limit it to this file?

Reply:
I think `colors.scss` would make sense 👍

Reply:
Cool! I fixed in 431388873810fcdaf39b9ab6fc3e2417a190de59, along with tweaking a color to match other headers on page from WP Core admin colors.